src/patternfly/components/Nav/nav.scss

```
@@ -135,9 +135,36 @@
     flex-grow: 1;
   }
 
+  // stylelint-disable max-nesting-depth
   .#{$menu} {
     --#{$menu}--MinWidth: 100%;
+
+    &.pf-m-drilldown {
+      --#{$menu}--PaddingBlockStart: 0;
+      --#{$menu}--PaddingBlockEnd: 0;
+      --#{$menu}--BackgroundColor: var(--#{$nav}--BackgroundColor);
+      --#{$menu}--BoxShadow: none;
+      --#{$menu}__list-item--Color: var(--#{$nav}__link--Color);
+      --#{$menu}__list-item--BackgroundColor: transparent;
+      --#{$menu}__list-item--hover--BackgroundColor: var(--#{$nav}__link--hover--BackgroundColor);
+      --#{$menu}__list-item--hover--BackgroundColor: transparent;
+      --#{$menu}__item--BackgroundColor: var(--#{$nav}__link--BackgroundColor);
+
+      .#{$menu}__list {
+        gap: var(--#{$nav}__subnav--RowGap);
```

Just so we don't need to change this if we ended up adding a column-gap for some reason

```suggestion
        row-gap: var(--#{$nav}__subnav--RowGap);
```